### PR TITLE
WIP: Use inline menu for showing bot commands

### PR DESCRIPTION
**Pull request by drondin**
Menu Screenshot
image

Setting value screenshots
Screenshot_2021-02-09 Telegram Web

image

### CODE REVIEW

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java [hunk 1]
```diff
@@ -68,33 +68,71 @@
...
-    public static String responseForCommand(String command) {
+    public static UserResponse responseForCommand(String command) {
         if (command == null) {
-            return UNKNOWN_COMMAND;
+            return new UserResponse(UNKNOWN_COMMAND, null);
         }
         switch (command) {
             case FARM_CHANGE:
-                return FARM_CHANGE_DESC + "\n" + PERCENT_VALUE_CALLBACK;
+                InlineButton[] buttons_farmChange = {
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/Commands.java:77]: please, use the java name convention for all variables, like buttonsFarmChange 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Class%20names%20should%20be%20nouns,such%20as%20URL%20or%20HTML).

File: src/main/java/pro/belbix/tgnotifier/tg/InlineButton.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package pro.belbix.tgnotifier.tg;
+
+import lombok.Data;
+
+@Data
+public class InlineButton {
+    private String text = null;
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/InlineButton.java:7]: setup null not necessary, it is the default value for objects
make variables `final` if you fill it up only In constructor

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java (hunk 1 above)
Comment (drondin) [reply src/main/java/pro/belbix/tgnotifier/tg/Commands.java:77]: Changed

File: src/main/java/pro/belbix/tgnotifier/tg/Commands.java [hunk 3]
```diff
@@ -9,92 +9,119 @@
...
+    public final static String PS_APR_CHANGE = "PS APR Change in %";
     public final static String PS_APR_CHANGE_DESC = "Profit Share APR change notification, in %";
 
-    public final static String HARD_WORK_MIN = "/hard_work_min";
+    public final static String HARD_WORK_MIN = "HardWork Change in USD";
     public final static String HARD_WORK_MIN_DESC = "You will receive notifications about doHardWork calls "
         + "earned more USD than this value";
 
-    public final static String SUBSCRIBE_ON_ADDRESS = "/subscribe_on_addr";
+    public final static String SUBSCRIBE_ON_ADDRESS = "Adress Events";
```
Comment (belbix) [inline src/main/java/pro/belbix/tgnotifier/tg/Commands.java:42]: fix typo

**Pull request merged**